invenio_administration/assets/semantic-ui/js/invenio_administration/theme.js

```diff
@@ -55,3 +55,32 @@ $invenioMenu.on("keydown", (event) => {
     handleBurgerCloseClick();
   }
 });
+
+
+/* Expand and collapse side menu */
+
+const $toggleButton = $("button.side-menu-toggle");
+const $leftIcon = $("button.side-menu-toggle .left.icon");
+const $rightIcon = $("button.side-menu-toggle .right.icon");
+const $sidebarMenu = $("#admin-side-menu");
+const $mainContent = $("#admin-main-content");
+const mainContentClassList = $mainContent[0].classList.value;
```

Comment:
i am wondering, does it always return an array, even if you give it an ID?

Reply:
Hmm, it did in this case at least, which is why I needed to access it like that. But I can double check...

Reply:
not exactly but something like [that](https://www.quora.com/Is-there-a-way-for-jQuery-to-return-a-single-object-instead-of-an-array-of-objects-For-example-if-I-use-an-id-selector) apparently.

If we would want to reduce jquery i think it could be done without too much added complexity using the [classList api](https://www.w3schools.com/jsref/prop_element_classlist.asp) but not sure if that is a consideration at the moment.

Reply:
Will check it out!

Just for some context: The reason I used `classList` is that it returns the full list, with the multiple occurrences of some classes. With semantic ui, we need to add e.g. the 'wide' class several times to get the responsiveness for each screen, for instance `sixteen wide mobile twelve wide tablet eight wide computer`. The `removeClass()` and `addClass()` functions treats multiple occurrences as only _one_ class, i.e. using `$elem.addClass("sixteen wide mobile twelve wide tablet")` will only add the `wide` class once.

Reply:
I think it is not necessary for now